scripts/actions/mobskills/great_wheel.lua

```diff
@@ -20,7 +20,11 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.SLASHING, xi.mobskills.shadowBehavior.NUMSHADOWS_3)
 
     target:takeDamage(dmg, mob, xi.attackType.PHYSICAL, xi.damageType.SLASHING)
-    mob:resetEnmity(target)
+
+    if not target:getObjType() == xi.objType.TRUST then
```

Comment:
oh, wait, no this cant be done

Reply:
Technically it can be, but it leads to confusion (and why CI calls us out on it).  But easiest is ~=